frontend/src/services/chat.tsx

```
@@ -51,3 +52,17 @@ export const chatStatus = async (projectId: string) => {
+
+export const draft_with_ai = async (data: ChatDraftRequest) => {
+  try {
+    const response = await PostRequest<ChatResponse>(
+      `${chatApiUrl}/draft`,
+      { ...data },
+      {},
+      300000
+    );
+    return response.data.data;
+  } catch (error) {
+    throw error;
```

Biome

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)